### ISSUE TITLE
WinForms - App Crash on .NET5 ARM64 when source is set - NullReferenceException

### ISSUE THREAD

**Issue by snickler** [MicrosoftEdge/WebViewFeedback#438]
**Description**
On Windows ARM64, the latest prerelease crashes at start with a NullReferenceException when the source is set. It's as simple as just adding the control to the form. It works from my machine running/targeting x64. Both utilizing the same exact .NET 5 Preview 8 build and same build of Windows 10.  I've also tested this with the latest .NET 5 RC1 build with the same results. 

```
Application: WebView2WindowsFormsBrowser.exe
CoreCLR Version: 5.0.20.40711
.NET Version: 5.0.0-preview.8.20407.11
Description: The process was terminated due to an unhandled exception.
Exception Info: System.NullReferenceException: Object reference not set to an instance of an object.
   at Microsoft.Web.WebView2.WinForms.WebView2.OnVisibleChanged(EventArgs e)
   at System.Windows.Forms.Control.AssignParent(Control value)
   at System.Windows.Forms.Control.ControlCollection.Add(Control value)
   at System.Windows.Forms.Form.ControlCollection.Add(Control value)
   at WebView2WindowsFormsBrowser.BrowserForm.InitializeComponent() in C:\Repos\WebView2Samples\SampleApps\WebView2WindowsFormsBrowser\BrowserForm.Designer.cs:line 217
   at WebView2WindowsFormsBrowser.BrowserForm..ctor() in C:\Repos\WebView2Samples\SampleApps\WebView2WindowsFormsBrowser\BrowserForm.cs:line 15
   at WebView2WindowsFormsBrowser.Program.Main() in C:\Repos\WebView2Samples\SampleApps\WebView2WindowsFormsBrowser\Program.cs:line 19

```

**Version**
SDK: 0.9.628-prerelease
Runtime: Canary 87.0.640.0 / Dev 87.0.634.0
Framework: WinForms
OS: Win10 10.0.20211.1000

**Repro Steps**

1. Clone https://github.com/MicrosoftEdge/WebView2Samples/
1. Modify /SampleApps/WebView2WindowsFormsBrowser/WebView2WindowsFormsBrowser.csproj and add net5.0 to TargetFrameworks
1. Run `dotnet run --framework net5.0`

****Expected****
Application running with WebView2 browser window.

****Actual****
Application crashes before rendering Form

**Comment by champnic** [comment]
Probably a duplicate (or related) to #431. We're working on a fix now!

**Comment by snickler** [comment]
@champnic - Actually I just tried something after reading that issue.

Running with 

`dotnet run -r win-x86 --framework net5.0`

WORKS on ARM64! 🤔🤔. 


![image](https://user-images.githubusercontent.com/4016293/93405831-1585ee80-f85c-11ea-94c9-e18dd7830f9c.png)

**Comment by champnic** [comment]
Glad you got it working! I'm going to close this out - the core issue will be fixed as part of #431. Let me know if you are running into unrelated errors and I can reopen. Thanks!
